test/fixtures/benchmarks.yml

```diff
@@ -4,4 +4,4 @@ one:
   ref_id: xccdf_org.ssgproject.content_benchmark_RHEL-7
   title: Benchmark One
   description: The first benchmark
-  version: v0.1.43
+  version: 0.1.43
```

Comment:
Nice catch. We could add a format validation on `version`. I think it'd be a good idea (in a different PR).